src/utils/bitreader.h

```diff
@@ -2,13 +2,15 @@
 #define BITREADER
 #include <stdint.h>
```

Comment:
`stdint.h` er deprecated i C++, brug `cstdint` i stedet for.

Reply:
fixed